umbraco/Umbraco-CMS
# Feature: Property label on top

This Feature enables all Properties to have their label displayed on top.
This is configured on the Property level, meaning it's not part of the DataType config.

**Test notes:**
* Check that any(a few) property editors can be used with Label On Top set to true.
* And see that the label goes on top of the property in the Content Editor.
* Check that it scales properly, label should go on top for all on small screens.

---
_This item has been added to our backlog [AB#8131](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/8131)_
